Application/EdFi.Ods.WebApi/appsettings.json

```diff
@@ -13,6 +13,7 @@
   },
   "AllowedHosts": "*",
   "BearerTokenTimeoutMinutes" : "30",
+  "SecurityMetadataCacheTimeoutMinutes": "0",
   "DefaultPageSizeLimit" : 500,
   "ApiSettings": {
     "Mode": "sandbox",
```

Comment:
The OdsTokens key below doesn't seem to be used, and didn't exist in the previous release of the API.  Can it be removed as part of this?  Its also set up as an array here, but in the generated development it outputs as empty string.  Probably need to remove from both.

Reply:
It looks like this is primarily used for deployment to drive which databases need to be generated.  However, this should be set to an empty string and not an array, the powershell is expecting it to be a semicolon separated string at the moment.

Reply:
updated to empty string